src/components/Admin/ManageSensors/ManageSensors.tsx

```diff
@@ -351,4 +351,5 @@ const ManageSensors: () => JSX.Element = () => {
   }
 };
 
+export type {Sensor};
```

Comment:
Maybe also move the `Sensor` interface into the Util file in the ManageSensors folder